### PR TITLE
ArtworkBanner for auctions should show sale cover image instead of partner profile image

### PR DESCRIPTION
**Pull request by oxaudo**
Address https://artsyproduct.atlassian.net/browse/DISCO-530

Huge props to @ds300 for simplifying relay mocking here. Unfortunately this still has our original issue here that fields that are redefined for different Contexts come out undefined, but at least the set up here is much simpler. cc @alloy - who originally helped me to figure out this mockResolver.

screen shot 2019-01-16 at 4 57 52 pm

### CODE REVIEW

File: src/Apps/Artwork/Components/ArtworkBanner/__tests__/ArtworkBanner.test.tsx [hunk 1]
```diff
@@ -22,17 +21,7 @@ describe("ArtworkBanner", () => {
...
-          ...response,
-          // FIXME: Need to figure out how to return proper values for aliesed objects
-          context: (_source, _args, _context, info: GraphQLResolveInfo) => {
-            const alias = info.fieldNodes[0].alias!.value
-            const x = response[alias]
-            return x
-          },
-        }),
-      },
+      mockData: { artwork: response },
```
Comment (damassi) [inline src/Apps/Artwork/Components/ArtworkBanner/__tests__/ArtworkBanner.test.tsx]: 👏

**Comment by damassi** [comment]
👍

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit d4bde92c288c0bb41664d438e7a64c847724b0e6

https://deploy-preview-1902--artsy-reaction.netlify.com

File: src/Apps/Artwork/Components/ArtworkBanner/__tests__/ArtworkBanner.test.tsx (hunk 1 above)
Comment (alloy) [reply src/Apps/Artwork/Components/ArtworkBanner/__tests__/ArtworkBanner.test.tsx]: Noice! ✨

Comment (oxaudo) [reply src/Apps/Artwork/Components/ArtworkBanner/__tests__/ArtworkBanner.test.tsx]: @alloy - unfortunately it still did not fix the issue with data with the same name for different contexts is not passed properly. Therefore there are still couple of line commented out in specs here. If you ever have time to look into that - would be awesome!

Comment (alloy) [reply src/Apps/Artwork/Components/ArtworkBanner/__tests__/ArtworkBanner.test.tsx]: Let's pair shortly tomorrow so I’m up to date on the current state.

**Pull request merged**